content/en/docs/tasks/traffic-management/egress/proxy/index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Istio as a Proxy for External Services
+description: Describes how to configure ingress gateway as a proxy for external services.
+weight: 9
+keywords: [traffic-management,ingress,https,http]
+---
+
+The [Control Ingress Traffic](/docs/tasks/traffic-management/ingress) and the
+[Ingress Gateway without TLS Termination](/docs/tasks/traffic-management/ingress/ingress-sni-passthrough/) tasks describe
+how to configure an ingress gateway to expose services inside the mesh to external traffic. The services can be HTTP or
```

Comment:
Since this is more focused on ingress gateway, it seems that the task belongs under `/docs/tasks/traffic-mangement/ingress` instead of `egress`. That said, it seems like a pretty specialized scenario so I wonder if it belongs under Tasks at all. Maybe it would be more suitable for a blog, or if we can finally come up with the exact criteria for what qualifies an an example, then maybe it should be an example?

Reply:
@frankbu I am OK with any location.

Reply:
@frankbu I rewrote the example as a blog post.